ml_peg/analysis/molecular/Wiggle150/analyse_Wiggle150.py

```diff
@@ -0,0 +1,216 @@
+"""Analyse Wiggle150 benchmark."""
+
+from __future__ import annotations
+
+from pathlib import Path
+import shutil
+
+from ase.io import read
+import pytest
+
+from ml_peg.analysis.utils.decorators import build_table, plot_parity
+from ml_peg.analysis.utils.utils import mae
+from ml_peg.app import APP_ROOT
+from ml_peg.calcs import CALCS_ROOT
+from ml_peg.models.get_models import get_model_names
+from ml_peg.models.models import current_models
+
+MODELS = get_model_names(current_models)
+CALC_PATH = CALCS_ROOT / "molecular" / "Wiggle150" / "outputs"
+OUT_PATH = APP_ROOT / "data" / "molecular" / "Wiggle150"
+
+WIGGLE150_THRESHOLDS = {"MAE": (0, 20.0)}
+
+
+def _sorted_xyz_files(model_dir: Path) -> list[Path]:
+    """
+    Return `.xyz` files for a model sorted by filename index.
+
+    Parameters
+    ----------
+    model_dir : Path
+        Directory containing per-model Wiggle150 outputs.
+
+    Returns
+    -------
+    list[Path]
+        Sorted list of `.xyz` files.
+    """
+    return sorted(model_dir.glob("*.xyz"), key=lambda path: int(path.stem))
+
+
+def _extract_metadata() -> tuple[list[str], list[str]]:
```

Comment:
I would maybe have `METADATA = _extract_metadata()`, and maybe return it as a dict, just so it can be accessed later without re-reading everything